components/rankings/rankingsUtils.js

```diff
@@ -32,6 +35,11 @@ const isValidMonth = async month => {
   return isValid.test(month)
 }
 
+const isValidYear = async year => {
```

Comment:
E aqui basta remover o async que essa função irá retornar apenas um true ou falze
```suggestion
const isValidYear = year => {
```